### PR TITLE
fix get-kube-local.sh 2 wrong parameters.

### PR DESCRIPTION
**Pull request by isaacdong**
**bug fix**
1. run ./get-kube-local.sh, and get this error.
>  ./get-kube-local.sh: line 60: 2: unbound variable
2. remove duplicate filename in docker run command: `--kubeconfig=${KUBELET_KUBECONFIG}`

**Special notes for your reviewer**:
code detail:
line 88: create-kubelet-kubeconfig "${KUBELET_KUBECONFIG}" 
line 59: function create-kubelet-kubeconfig() {
line 60:  local destination="${1}"

**Release note**:

```release-note
      NONE
```

### CODE REVIEW

**Comment by roberthbailey** [comment]
Other implementations of the `create-kubelet-kubeconfig` function take an apiserver address as the first argument and the destination as the second argument. Having dest as $2 is consistent with those. 

It might make more sense to fix the call site to be
```bash
create-kubelet-kubeconfig "http://localhost:8080" $KUBELET_KUBECONFIG
```
instead of making the function signature diverge from the other functions of the same name. WDYT?

**Comment by isaacdong** [comment]
There are three funtion create-kubelet-kubeconfig in code base:
https://github.com/kubernetes/kubernetes/search?q=create-kubelet-kubeconfig&unscoped_q=create-kubelet-kubeconfig

Keep them consistent.
Good idea, PR updated.

File: cluster/get-kube-local.sh [hunk 1]
```diff
@@ -55,8 +55,9 @@ function run {
 }
 
 # Creates a kubeconfig file for the kubelet.
-# Args: destination file path
+# Args: the IP address of the API server (e.g. "http://localhost:8080"), destination file path
 function create-kubelet-kubeconfig() {
+  local api_addr="${1}"
```
Comment (roberthbailey) [inline cluster/get-kube-local.sh]: this var isn't used (the api addr is hard coded to localhost:8080 on line 75). You should either use the var or comment that it is ignored.

**Review by roberthbailey — COMMENTED** [review]
Can you please squash your changes?

**Pull request merged**